webapp/static/interop-data.json

```diff
@@ -322,119 +322,118 @@
+                "spec": "https://w3c.github.io/csswg-drafts/css-backgrounds/#the-border-image",
+                "tests": "/results/?label=experimental&label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2023-cssborderimage",
                 "countsTowardScore": true
             },
             "interop-2023-color": {
                 "description": "Color Spaces and Functions",
-                "mdn": "",
-                "spec": "",
+                "mdn": "https://developer.mozilla.org/en-US/docs/Web/CSS/color_value",
+                "spec": "https://w3c.github.io/csswg-drafts/css-color/#color-syntax",
                 "tests": "/results/?label=master&label=experimental&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2022-color%20or%20label%3Ainterop-2023-color", 
                 "countsTowardScore": true
             },
             "interop-2023-container": {
                 "description": "Container Queries",
-                "mdn": "",
-                "spec": "",
-                "tests": "/results/?label=experimental&label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2023-container", 
+                "mdn": "https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Container_Queries",
+                "spec": "https://www.w3.org/TR/css-contain-3/#container-queries",
+                "tests": "/results/?label=experimental&label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2023-container",
                 "countsTowardScore": true
             },
             "interop-2023-contain": {
                 "description": "Containment",
-                "mdn": "",
-                "spec": "",
+                "mdn": "https://developer.mozilla.org/docs/Web/CSS/contain",
+                "spec": "https://drafts.csswg.org/css-contain/#contain-property",
                 "tests": "/results/css?label=master&label=experimental&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2022-contain%20or%20label%3Ainterop-2023-contain", 
                 "countsTowardScore": true
             },
             "interop-2023-pseudos": {
                 "description": "CSS Pseudo-classes",
-                "mdn": "",
-                "spec": "",
-                "tests": "/results/?label=experimental&label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2023-pseudos", 
+                "mdn": "https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-classes",
+                "spec": "https://html.spec.whatwg.org/multipage/semantics-other.html#pseudo-classes",
+                "tests": "/results/?label=experimental&label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2023-pseudos",
```

Comment:
Would be cool if we linked directly to the css/selectors directory, since no test is outside of that dir.